backend_tv/adjust_src.cpp

```diff
@@ -297,15 +289,6 @@ void checkSupport(Instruction &i) {
       *out << "\nERROR: llvm.thread instrinsics not supported\n\n";
       exit(-1);
     }
-    if ((callee.find("llvm.experimental.gc") != string::npos) ||
```

Comment:
wait, I don't think this code was duplicated!

Reply:
Ah my bad. I'll fix it. I may have accidentally deleted that one.